plugins/omemo/src/ui/contact_details_dialog.vala

```diff
@@ -48,7 +48,7 @@ public class ContactDetailsDialog : Gtk.Dialog {
         automatically_accept_new_descr.label = _("When this contact adds new encryption keys to their account, automatically accept them.");
         own_key_label.label = _("Own key");
         new_keys_label.label = _("New keys");
-        associated_keys_label.label = _("Associated keys");
```

Comment:
Associated with what? I think when there's first "new keys", then "other keys" should be clear (i.e. they're not new)